tests/system/system_integration.txt

```diff
@@ -0,0 +1,503 @@
+test_marathon_basics.py:def test_launch_mesos_container():
```

Comment:
what is this? :)

Reply:
This is some awesome stuff which should not be checked in.